src/main/java/com/yandex/money/api/typeadapters/model/showcase/ComponentsTypeProvider.java

```diff
@@ -27,17 +27,8 @@
 import com.yandex.money.api.model.showcase.components.Component;
 import com.yandex.money.api.model.showcase.components.containers.Group;
 import com.yandex.money.api.model.showcase.components.containers.Paragraph;
-import com.yandex.money.api.model.showcase.components.uicontrols.Amount;
-import com.yandex.money.api.model.showcase.components.uicontrols.Checkbox;
-import com.yandex.money.api.model.showcase.components.uicontrols.Date;
-import com.yandex.money.api.model.showcase.components.uicontrols.Email;
-import com.yandex.money.api.model.showcase.components.uicontrols.Month;
+import com.yandex.money.api.model.showcase.components.uicontrols.*;
```

Comment:
Fully qulify imports: https://source.android.com/setup/contribute/code-style#fully-qualify-imports

Reply:
Also, changes in this file are unnecessary.